src/templates/sample.ts

```diff
@@ -18,18 +18,21 @@ export const templateCertificate: TemplateCertificate = {
 };
 
 export interface TemplateCertificateV3 extends v3.OpenAttestationDocument {
-  recipient: { name: string };
+  credentialSubject: {
+    recipient: { name: string };
+  };
 }
 
 export const templateCertificateV3: TemplateCertificateV3 = {
   "@context": [],
-  recipient: { name: "John Doe" },
   issuer: {
     name: "Name",
     id: "id"
   },
   type: "Credential",
-  credentialSubject: {},
+  credentialSubject: {
+    recipient: { name: "John Doe" }
+  },
```

Comment:
```suggestion
  credentialSubject: {
    name: "John Doe"
  },
```

- [credentialSubject](https://www.w3.org/TR/vc-data-model/#credential-subject).
  - Single subject [reference](https://www.w3.org/TR/vc-data-model/#example-usage-of-the-credentialsubject-property).
  - Multiple subjects [reference](https://www.w3.org/TR/vc-data-model/#example-specifying-multiple-subjects-in-a-verifiable-credential).

Single subject should be the same, with `name` referring to the actual recipient of VC?